src/RealAntennasProject/RealAntenna.cs

```diff
@@ -27,8 +27,9 @@ public class RealAntenna
         public virtual double Bandwidth => DataRate;          // RF bandwidth required.
         public virtual float AMWTemp { get; set; }
         public virtual float Beamwidth => Physics.Beamwidth(Gain);
+        public string encoderOverride {  get; set; }
```

Comment:
You should match case with the rest of the code. Also I spot an extra whitespace.

Reply:
I tried to match the formatting (although I have no idea what `virtual` means, it still works fine) and I think I removed the whitespace in question.